packages/web-components/src/dropdown/README.md

```diff
@@ -0,0 +1,176 @@
+| ----------------- | ------- | ---------------------------------------------------------------------------------------------------------------- | --------- | ---------------------------------------------------- |
+| `appearance`      | public  | `{ outline: 'outline', underline: 'underline', filledDarker: 'filled-darker', filledLighter: 'filled-lighter' }` | `outline` | Sets style variations.                               |
+| `cleanup`         | public  | `() => void`                                                                                                     |           | Cleanup function for the listbox positioner.         |
+| `disabled`        | public  | `boolean`                                                                                                        |           | The disabled state of the listbox.                   |
+| `displayValue`    | public  | `string`                                                                                                         |           | The value displayed on the button.                   |
+| `length`          | public  | `number`                                                                                                         |           | The number of options.                               |
+| `multiple`        | public  | `boolean`                                                                                                        |           | Indicates if the listbox is in multi-selection mode. |
+| `open`            | public  | `boolean`                                                                                                        | `false`   | The open attribute.                                  |
+| `options`         | public  | `FlunetListboxOption[]`                                                                                          |           | The list of options.                                 |
+| `proxy`           | public  |                                                                                                                  |           |                                                      |
+| `selectedIndex`   | public  | `number`                                                                                                         | `-1`      | The index of the selected option.                    |
+| `selectedOptions` | public  | `FluentListboxOption[]`                                                                                          | `[]`      | A collection of the selected options.                |
+| `size`            | public  | `{ small: 'small', medium: 'medium', large: 'large' }`                                                           | `medium`  | The maximum number of options to display.            |
```

Comment:
This is likely going to be a problem as "size" is an attribute inherited from FASTSelect which is *required* for multiple. How will we reconcile the _size_ attribute being intended to detail the number of options shown with the visual "size" in FUIR9?

Reply:
For the record, discussed in other PRs, we will use `control-size`.